ietf/status/views.py

```diff
@@ -0,0 +1,50 @@
+# Copyright The IETF Trust 2016-2020, All Rights Reserved
+# -*- coding: utf-8 -*-
+
+import json
+from django.http import HttpResponse, HttpResponseRedirect, HttpResponseNotFound
+from ietf.utils import markdown
+from django.core.exceptions import FieldError
+from django.shortcuts import render, get_object_or_404
+from ietf.status.models import Status
+
+import debug                            # pyflakes:ignore
+
+def get_context_data():
+    status = Status.objects.order_by("-date").first()
+    if status is None or status.active == False:
+        return { "hasMessage": False }
+
+    if status.slug is None:
+        raise FieldError("No slug generated. This shouldn't happen.")
```

Comment:
Not sure this is a valid use of FieldError. In any case there's nothing to catch this, so it would result in a 500 being served.

It would be better to make the slug a required and unique field in the model with `null=False, blank-False, unique=True`, with  and let the ORM ensure it will never be empty,